### PR TITLE
{vis}[GCCcore/10.2.0] JasPer v2.0.24

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

The sources of JasPer moved to a GitHub repo

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node354.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/b15a4c21d25b81ffaa41d1852258277d for a full test report.

File: easybuild/easyconfigs/j/JasPer/JasPer-2.0.24-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ('CMake', '3.18.4'),
+]
+
+configopts = '-DJAS_ENABLE_DOC=OFF '
+
+sanity_check_paths = {
+    'files': ['bin/jasper', ('lib/libjasper.%s' % SHLIB_EXT, 'lib64/libjasper.%s' % SHLIB_EXT)],
+    'dirs': ['include'],
+}
+
```
Comment (smoors) [inline easybuild/easyconfigs/j/JasPer/JasPer-2.0.24-GCCcore-10.2.0.eb:33]: @lexming maybe add a sanity_check_commands (--help exits with non-zero)?
```suggestion
sanity_check_commands = ['jasper --version']
```

Comment (lexming) [reply easybuild/easyconfigs/j/JasPer/JasPer-2.0.24-GCCcore-10.2.0.eb]: sure :+1:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node375.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/20e6db64a8b8ab2ed0c6781e758d5d2a for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3547.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/37d61d7412bda8921cda24275485af3b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2691.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/00b70c90977df339436fc7e7e2a83dd9 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @lexming!

**Pull request merged**